app/lib/synthetic_study_populator.rb

```diff
@@ -19,20 +20,26 @@ def self.populate_all(user: User.first)
 
   # populates the synthetic study specified in the given folder (e.g. ./db/seed/synthetic_studies/blood)
   # destroys any existing studies and workspace data corresponding to that study
-  def self.populate(synthetic_study_folder, user: User.first, detached: false, update_files: false)
-    synthetic_study_path = synthetic_study_folder
-    if (synthetic_study_folder.exclude?('/'))
-      synthetic_study_path = DEFAULT_SYNTHETIC_STUDY_PATH.join(synthetic_study_folder).to_s
+  def self.populate(study_folder, user: User.first, detached: false, update_files: false)
+    tudy_path = study_folder
```

Comment:
Blocker: this looks like a bug.

```suggestion
    study_path = study_folder
```

Reply:
yup, good catch